homeassistant/components/formula_one/translations/en.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
We don't keep translations in our repository anymore. Please git rebase this PR onto the latest development branch and remove this file.